addons/diagnostic/fnc_watchStop.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Internal Function: CBA_diagnostic_fnc_watchStop
+
+Description:
+    Stop watching an input
+
+Parameters:
+    _index - The index in the watch list
+
+Returns:
+    nil
+
+Examples:
+    (begin example)
+        [1] call CBA_diagnostic_fnc_watchStop;
+    (end)
+
+Author:
+    Brett Mayson
+---------------------------------------------------------------------------- */
+
+params ["_index"];
+
+private _ctrl = GVAR(watchControls) select _index;
```

Comment:
When you run `watchStart` for the first time it runs `watchStop`, `_ctrl ` will be nil in that case. Does not matter much in unscheduled as it will simply skip the `if` but I feel like it's cleaner with `param`.
```suggestion
private _ctrl = GVAR(watchControls) param [_index, controlNull];
```